homeassistant/components/roon/media_player.py

```diff
@@ -491,8 +486,8 @@ def play_media(self, media_type, media_id, **kwargs):
                     path_list,
                 )
 
-    def join(self, join_ids):
-        """Add another Roon player to this player's join group."""
+    async def async_join_players(self, group_members):
```

Comment:
Is the work in the method async safe?

Reply:
Hmm - probably not - I'll switch them both.